dashboard/src/utils/filterDataset.js

```diff
@@ -1,7 +1,6 @@
-import {dateFromUTCString } from "./constructDate";
 export const filterData = (dataArray, startDate, endDate, controllerValue) => {
   let res = dataArray.filter((data) => {
-    let datasetDate = dateFromUTCString(`${data.metadata["dataset.created"]}`);
+    let datasetDate = new Date(`${data.metadata["dataset.created"]}`)
```

Comment:
Shouldn't the `;` still be here?

Also, the backtick string seems pointless since the entire string is just the value you're substituting. It'd be easier to read without that:

```
  let datasetDate = new Date(data.metadata["dataset.created"]);
```